### PR TITLE
Ban all players from playing /tg/station

### PR DESCRIPTION
**Pull request by JaytleBee**
## About The Pull Request

This will prevent players from connecting to the /tg/station server

## Why It's Good For The Game

/tg/station has brought misery into the lives of many. It may very well be one of the worst time-sinks of all time, the least satisfying game to ever exist and also horribly coded. Merging this will put the beast that is the codebase out of its misery and let the sorry souls who play this game (myself included) finally move on with their life.

## Changelog
:cl:
add: Base functionality
/:cl:

### CODE REVIEW

**Comment by KingTerrytheTerrible** [comment]
ready to merge

**Comment by HomocidlePancakes** [comment]
My sister was addicted to heroine. We had to take some extreme measures in order to help her. I always thought that I was better than her, and that only weak people get addicted. But two years ago I discovered SS13 and now I understand what a sad human being I am. This game has brought to me only suffering and misery, and yet, I can’t stop playing. I just FUCKING CAN’T! I NEED IT! I DON’T EVEN ENJOY IT, I JUST NEED STIMULI. PLEASE, SOMEBODY, HELP! PUT ME OUT OF MY MISERY! KILL ME!

**Comment by XDTM** [comment]
if this doesn't work just add an in-game coding tutorial, everyone knows coders don't play

**Comment by plapatin** [comment]
finally some good fucking code

**Comment by SpinnerMaster** [comment]
speedmerge

**Comment by TheDanseMacabre** [comment]
based pr

**Comment by cacogen** [comment]
not sure why this was closed

File: code/modules/client/client_procs.dm [hunk 1]
```diff
@@ -209,244 +209,11 @@ GLOBAL_LIST_EMPTY(external_rsc_urls)
...
-		var/datum/verbs/menu/menuitem = GLOB.menulist[thing]
-		if (menuitem)
-			menuitem.Load_checked(src)
-
-	Master.UpdateTickRate()
+	
+	. = ..()
+	to_chat(src, "<span class='userdanger'>You are banned from playing /tg/station.</span>")
+	qdel(src)
+	return
```
Comment (PKPenguin321) [inline code/modules/client/client_procs.dm]: Trailing return

Comment (JaytleBee) [reply code/modules/client/client_procs.dm]: No

**Pull request closed without merging**